velox/functions/lib/tests/Re2FunctionsTest.cpp

```diff
@@ -56,12 +56,30 @@ class Re2FunctionsTest : public test::FunctionBaseTest {
   }
 
  protected:
+  const char* kAlphaNumCharSet =
```

Comment:
Can there be any other special characters in this set like @, # etc ?

Reply:
Thanks for pointing these combinations, added other possible characters to the set